transports/webrtc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-webrtc"
-version = "0.4.0-alpha.2"
+version = "0.4.0-alpha.3"
```

Comment:
Do I understand correctly that one can import this new `libp2p-webrtc` version through the `v0.51.0` `libp2p` meta crate, as the `v0.51.0` `libp2p` meta crate does not pin a specific version, more specifically `0.4.0-alpha.2` (note that there is no `=`)?

https://github.com/libp2p/rust-libp2p/blob/v0.51.0/Cargo.toml#L126

Should we release `v0.51.1` of the `libp2p` meta crate, pinning all the alpha releases, like we did for `v0.50.1`?  Similar to https://github.com/libp2p/rust-libp2p/pull/3540? Otherwise we will get into the same trouble once we release a breaking new alpha of `libp2p-{quic,tls,webrtc}`, right?

Reply:
Yes. I thought that we did pin those? Did my PR for moving the code into the libp2p directory mess this up?

Reply:
We do alphas correctly as per the semver spec. Given  that pre-releases allow for breaking changes,  one could argue that cargo should pin them by default and not upgrade from one to the other.

I am not sure if this has ever been considered.

Reply:
Any objections to the above suggestion? As far as I can tell it solves all our issues.

- No more accidental automatic upgrade, unless intended (patch version bump).
- No issues with transient dependencies, e.g. having to pin `libp2p-tls` in both `libp2p-quic` and `libp2p` where `libp2p-quic` is as well an alpha, thus needing a bump, thus needing a bump in `libp2p`, ...

Reply:
I think my preference would be to remove the dependencies instead and have users deal with it or we just drop the pre-release entirely.

Reply:
> > and have users deal with it or we just drop the pre-release entirely.
> 
> How would they deal with a breaking change in `libp2p-tls`? Would they pin it even though it is a transient dependency? The above suggestion solves this issue.

That is still one we'd have to pin within `libp2p-quic` but I think pinning alpha's within other alpha's is simply something that you have to accept?

> > I think my preference would be to remove the dependencies instead
> 
> I am in favor of removing `libp2p-quic` and `libp2p-webrtc` from the meta crate.

`libp2p-tls` too right? Or should we promote that one as non-alpha? To be honest, I don't see why it would be alpha, we are depending on `rustls` there which is quite tested already, no?